migrations/versions/750e1f83b191_add_global_registrant_to_permissions.py

```diff
@@ -0,0 +1,33 @@
+"""Add global_registrant to permissions.
+
+Revision ID: 750e1f83b191
+Revises: ba020591fae8
+Create Date: 2019-06-27 07:38:22.728758
+
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sqlalchemy as sa
+from alembic import op
+
+# Revision identifiers, used by Alembic.
+revision = '750e1f83b191'
+down_revision = u'ba020591fae8'
```

Comment:
(Nitpicking, but given the `unicode_literals` import, the `u` is redundant. I would assume this is partly generated as well, so maybe a discrepancy in that tool. No need to fix this, just noting.)

Reply:
It's generated